docs/source/users/index.md

```diff
@@ -342,14 +344,29 @@ providers, as defined in [`langchain.llms`](https://langchain.readthedocs.io/en/
 - `openai-chat`
 - `sagemaker-endpoint`
 
-You can find a list of supported providers and models by running `%ai list`. Some providers
-define a list of supported models. If a provider does not define a list of supported models,
-consult the vendor's documentation. The [Hugging Face web site](https://huggingface.co/)
-includes a list of models, for example.
+:::{warning}
+As of v0.8.0, only the `%%ai` *cell* magic may be used to invoke a language
+model, while the `%ai` *line* magic is reserved for invoking subcommands.
+:::
+
+### Listing available models
+
+Jupyter AI also includes multiple subcommands, which may be invoked via the
+`%ai` *line* magic. Jupyter AI uses subcommands to provide additional utilities
+in notebooks while keeping the same concise syntax for invoking a language model.
+
+The `%ai list` subcommand prints a list of available providers and models. Some
+providers explicitly define a list of supported models in their API. However,
+other providers, like HuggingFace Hub, lack a well-defined list of available
+models. In such cases, it's best to consult the provider's upstream
+documentation. The [HuggingFace website](https://huggingface.co/) includes a
+list of models, for example.
```

Comment:
```suggestion
The `%ai list` subcommand prints a list of available providers and models. Some
providers provide a list of supported models, and others do not.
If you don't see a list, consult the provider's documentation. The
[HuggingFace website](https://huggingface.co/) includes a
list of models, for example.
```

Reply:
This is a good callout for a larger documentation issue that I believe should be tackled in a separate PR. I don't like the fact that we are not exposing how the model ID is determined at all in the help documentation, and that we are exclusively forwarding them to a provider's upstream documentation. This is seldom helpful, as it doesn't offer guidance on how the local model ID is to be specified. I've created an issue to track this: https://github.com/jupyterlab/jupyter-ai/issues/198